Rudder.podspec

```diff
@@ -12,7 +12,7 @@ Pod::Spec.new do |s|
 
   s.homepage         = "https://github.com/rudderlabs/rudder-sdk-ios"
   s.license          = { :type => "Apache", :file => "LICENSE" }
-  s.author           = { "RudderStack" => "arnab@rudderlabs.com" }
+  s.author           = { "RudderStack" => "sdk-accounts@rudderstack.com" }
```

Comment:
sdk-accounts is the user for third party tools accounts. sdk@rudderstack.com is the one for the team group account

Reply:
Updated.